packages/@aws-cdk/custom-resource-handlers/lib/aws-s3-deployment/bucket-deployment-handler/index.py

```diff
@@ -135,7 +136,7 @@ def cfn_error(message=None):
         cfn_send(event, context, CFN_SUCCESS, physicalResourceId=physical_id, responseData={
             # Passing through the ARN sequences dependencees on the deployment
             'DestinationBucketArn': props.get('DestinationBucketArn'),
-            'SourceObjectKeys': props.get('SourceObjectKeys'),
+            **({'SourceObjectKeys': props.get('SourceObjectKeys')} if output_object_keys else {})
```

Comment:
I'm not too familiar on what the `**` operator does in this context?

Reply:
This operator is to merge the content of that dictionary with the parent dictionary